processor/src/test/java/com/squareup/sqlbrite/SqlBriteProcessorTest.java

```diff
@@ -0,0 +1,42 @@
+package com.squareup.sqlbrite;
+
+import com.google.common.io.Files;
+import com.google.testing.compile.JavaFileObjects;
+
+import com.squareup.sqlbrite.annotation.Column;
+import com.squareup.sqlbrite.annotation.Table;
+
+import org.junit.Test;
+
+import android.content.ContentValues;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static com.google.common.truth.Truth.assert_;
+import static com.google.testing.compile.JavaSourceSubjectFactory.javaSource;
+
+public class SqlBriteProcessorTest {
+
+  @Test
+  public void testSanity() throws IOException {
+    File file = new File("src/test/java/com/squareup/sqlbrite/SqlBriteProcessorTest.java");
```

Comment:
I'll probably replace this with a bunch of success and failure tests before merging.